client/index.html

```diff
@@ -274,7 +274,7 @@ <h1 class="title">Settings</h1>
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
 									{{#each themes}}
-										<option value="{{filename}}">
+										<option value="{{url}}">
```

Comment:
Ah, gotcha. I guess we can just load the local themes with plugins/ in the URL, that's the nicest way to deal with this, I think. Keep the value as the name and they all have the same URL then.